tests/ssl-opt.sh

```diff
@@ -3115,7 +3115,7 @@ requires_config_enabled MBEDTLS_DEBUG_C
 run_test    "TLS 1.3: psk_key_exchange_modes: basic check, O->m" \
             "$P_SRV force_version=tls13 debug_level=5" \
             "$O_NEXT_CLI -tls1_3 -psk 6162636465666768696a6b6c6d6e6f70 -allow_no_dhe_kex" \
-            0 \
+            1 \
```

Comment:
I am not sure to understand why this test fails in the prototype that is more capable than development (@PR6078).

Reply:
For time being, `pre_shared_key`  is implemented in `prototype` and  it is not in PR6078 . The test should be fail in `prototype`.

With `pre_shared_key` , this test will report binder error.